plugins/modules/ecs_taskdefinition.py

```diff
@@ -627,6 +627,26 @@
             expression:
                 description: A cluster query language expression to apply to the constraint.
                 type: str
+    runtime_platform:
+        description:
+            - runtime platform configuration for the task
+        required: false
```

Comment:
Can you please add `version_added: 6.2.0` ?

Reply:
@romulus-ai

Reply:
done, delay was because of holiday